scripts/archlinux/solidity-git/PKGBUILD

```diff
@@ -0,0 +1,65 @@
+# Maintainer: Afri 5chdn <aur@cach.co>
+# Available from AUR: https://aur.archlinux.org/packages/solidity-git/
+
+pkgname=solidity-git
+pkgver=0.4.3.git.20161020.9d30450
```

Comment:
Please check out `script/travis/publish_binary.sh` for how to determine the accurate compiler version. It would be nice to use it here too.